lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -2,6 +2,14 @@ module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmCloud
     expose :cloud_networks, :association => true
 
+    expose :resize
```

Comment:
Right, resize needs parameter - flavor. It looked to me, that it is possible (https://github.com/ManageIQ/manageiq/blob/master/lib/miq_automation_engine/engine/miq_ae_service_model_base.rb#L122), but I can define a method instead of expose if needed - would that be acceptable?